### PR TITLE
Update the response for the deployment groups endpoint.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/481576376873776

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://12-bb-apidocs.surge.sh

File: deployment_groups/get-list.adoc [hunk 1]
```diff
@@ -96,11 +96,20 @@ $.ajax(settings).done(function (response) {
...
+    ],
+    "name": "Test Team",
+    "prefixes": [
+      {
+        "name": "myprefix"
+      }
+    ],
     "testers": [
-      "\peter@buddybuild.com"
+      "jeff@buddybuild.com",
```
Comment (jeffrey-ng) [inline deployment_groups/get-list.adoc]: We could probably change this back to \peter@buddybuild.com

File: deployment_groups/get-list.adoc [hunk 2]
```diff
@@ -96,11 +96,20 @@ $.ajax(settings).done(function (response) {
 [source,json]
 [
   {
-    "_id": "274a0fc14333e39be0aea9db",
-    "name": "The Team",
-    "branch": "master",
+    "_id": "5a03360e460f821150c5559b",
+    "branches": [
+      "IOS-12345",
```
Comment (jeffrey-ng) [inline deployment_groups/get-list.adoc]: master

File: deployment_groups/get-list.adoc [hunk 3]
```diff
@@ -96,11 +96,20 @@ $.ajax(settings).done(function (response) {
...
+      "tag:feature-release-3"
```
Comment (jeffrey-ng) [inline deployment_groups/get-list.adoc]: feature

File: deployment_groups/get-list.adoc [hunk 4]
```diff
@@ -96,11 +96,20 @@ $.ajax(settings).done(function (response) {
...
+    ],
+    "name": "Test Team",
```
Comment (jeffrey-ng) [inline deployment_groups/get-list.adoc]: The Team

**Comment by bbdocs** [comment]
See the website here: http://12-bb-apidocs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://12-bb-apidocs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://12-bb-apidocs.surge.sh

**Pull request merged**